src/App.css

```diff
@@ -7,27 +10,156 @@ ul {
   list-style: none;
 }
 
-/* ############### Genral Style ################# */
+button {
+  font-family: 'Roboto Slab', serif;
+  width: 11.5rem;
+  height: 3rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border: none;
+  border-radius: 5px;
+  background-color: #0290ff;
+  font-size: 0.9rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  cursor: pointer;
+}
+
+form {
+  width: 100%;
+  padding: 1.8rem 6.25rem;
+}
+
+/* ############### Styling ################# */
 .App {
   width: 100%;
 }
 
 #header {
+  font-family: 'Montserrat', sans-serif;
   width: 100%;
+  background-color: #fff;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+}
+
+.logo {
+  color: #0290ff;
+  font-size: 1.875rem;
+  font-weight: bold;
 }
 
-.container {
+.links {
   display: flex;
   flex-direction: row;
-  gap: 20px;
+  gap: 3rem;
 }
 
 .navbar {
-  width: 80%;
-  height: 80px;
-  margin: 0 auto;
+  height: 6rem;
+  padding: 0 6.25rem;
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
 }
+
+.navLinks {
+  display: flex;
```

Comment:
- [ ] Since you are using this role for multiple elements, it will be nice to use a `class attribute` instead as this will make your `CSS` dry. Please kindly fix that, as it is a requirement for this project and best practice.  Checkout this [article](https://speckyboy.com/good-bad-css-practices/) for more clarifications. See screenshot of the requirements below. 

![styling](https://user-images.githubusercontent.com/65852621/150609945-6ca349fc-a5ae-432c-85cb-d190d8869f01.png)

Reply:
Hello @Mihndim2020,

I didn't understand what you really mean by that comment.
You didn't point to something specific to change.
Can you explain more?

Reply:
Hello @Mihndim2020 , 
Please check if you have any misunderstandings as I'm not part of this project and the things you are pointing doesn't make too much sense.

Regards.

Reply:
Hello @Omar-Muhamad  the issue is with the fact the you have repeated `display: flex` in your `css file` several times. Which is not a good practice, so, you can give a class attribute say `d-flex` to all elements in your `html` file the has a `css style` of `display: flex`, then in your `css file`, just select the class and give it a display of flex like so `d-flex {display: flex}` thus making your `css file` dry. That is avoiding repetition.